Snake/Snake/Reader.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Threading;
+
+namespace Snake
+{
+    // This class is used as a key listner. It waits for a few milliseconds for a key to be pressed.

```

Comment:
then call it that. Reader tells me nothing. you should not need a comment like this if you give things better names